types/sitemap-generator/sitemap-generator-tests.ts

```diff
@@ -0,0 +1,46 @@
+import SitemapGenerator from 'sitemap-generator';
```

Comment:
```suggestion
import SitemapGenerator = require('sitemap-generator');
```
this will work without a need for `esModuleInterop`